src/sentry/integrations/pagerduty/utils.py

```diff
@@ -116,6 +123,16 @@ def build_incident_attachment(
     }
 
 
+def attach_custom_severity(data: dict[str, Any], action: AlertRuleTriggerAction) -> dict[str, Any]:
+    # use custom severity (overrides default in build_incident_attachment)
+    if action.sentry_app_config is None:
+        return data
+
+    severity = action.sentry_app_config.get("priority", None)
+    if severity is not None:
+        data["payload"]["severity"] = severity
```

Comment:
we have different default pagerduty severities depending on the incident status https://github.com/getsentry/sentry/blob/d6e50141786f2fe0103667d74f1955e018906213/src/sentry/integrations/pagerduty/utils.py#L100-L110